cmd/record.go

```diff
@@ -91,10 +115,6 @@ func (r *Record) GetCmd() *cobra.Command {
 			if err != nil {
 				r.logger.Error("Failed to get the command to run the user application", zap.Error((err)))
 			}
-
-			if appCmd == "" {
-				appCmd = confRecord.Command
-			}
 			
```

Comment:
@charankamarapu sorry couldn't understand you

Reply:
`r.GetRecordConfig(&path, &proxyPort, &appCmd, &appContainer, &networkName, &delay, &ports)` is placed after 

```
if appCmd == "" {
         fmt.Println("Error: missing required -c flag\n")
	 func (r *Record) GetCmd() *cobra.Command {
         r.logger.Error("Failed to get the application's docker container name", zap.Error((err)))
}
```
right, so first it will check appCmd == "" and execute that condition and throws error right..?

Reply:
Is this clear ..?

Reply:
Yeah thanks :)